CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
 # Release History
 
-## 1.0.0b1 (Unreleased)
+## 1.0.0b1 (05/22/2024)
+
+New Feature Management library.
+
+Provides the ability to manage features in a project. Enables:
+
+* Loading of feature flags from a file, see: https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v2.0.0.schema.json
```

Comment:
This definition references a feature flag definition that defines variants. Is that intended?

Reply:
This is the only one that has the prefix "feature_management.feature_flags".

Reply:
Got it, and that is the format that is required in this version of the feature management library, I suppose?

Reply:
Yes, it's the only one we support for this and in JS.

Reply:
Sounds good!